cis/__init__.py

```diff
@@ -1,7 +1,3 @@
 __all__ = [
```

Comment:
Either remove or fix this entry. This way when importing everything under module ``cis`` imports nothing.